lib/tools/mp4-inspector.js

```diff
@@ -445,14 +445,53 @@ var
             flags: new Uint8Array(data.subarray(1, 4)),
             references: [],
             referenceId: view.getUint32(4),
-            timescale: view.getUint32(8),
-            earliestPresentationTime: view.getUint32(12),
-            firstOffset: view.getUint32(16)
+            timescale: view.getUint32(8)
           },
-          referenceCount = view.getUint16(22),
-          i;
+        earliestPresentationTime,
+        firstOffset,
+        i = 12;
 
-      for (i = 24; referenceCount; i += 12, referenceCount--) {
+      if (result.version === 0) {
+        earliestPresentationTime = view.getUint32(i);
+        i += 4;
+        firstOffset = view.getUint32(i);
+        i += 4;
+      } else {
+        // read 64 bits
+        earliestPresentationTime = view.getUint32(i);
+        earliestPresentationTime *= Math.pow(2, 32);
```

Comment:
Regular integer in javascript is 52 bits. Might want to use [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) for 64 bits number.

Reply:
Unfortunately, browser compatibility doesn't allow us to use BigInt yet. However, since we're working with Uint32 array views, we should still be able to access the full value.